src/date-picker/date-picker.jsx

```diff
@@ -5,6 +5,15 @@ let DateTime = require('../utils/date-time');
 let DatePickerDialog = require('./date-picker-dialog');
 let TextField = require('../text-field');
 
+/**
+ * Check if a value is a valid Date instance.
+ *
+ * @param value The value to check.
+ * @returns {boolean} True if the object provided is valid, false otherwise.
+ */
+function isValid(value) {
+  return value instanceof Date;
+}
```

Comment:
This function can go in date-time.js and probably should be renamed to `isDateObject` - https://github.com/callemall/material-ui/blob/master/src/utils/date-time.js#L154